.github/workflows/ci.yml

```diff
@@ -154,15 +154,36 @@ jobs:
         env:
           CLUSTER_NAME: "dbaas-ci-20221005"
           NODEPOOL_NAME: "terraform-node-dbaas-ci-20221005"
+      - name: Clean up custom resources
+        if: always()
+        run: |
+          kubectl delete db --all-namespaces --all
+          kubectl delete pvc --all-namespaces --all
```

Comment:
Wouldn't this affect all other tests running in parallel?

Reply:
I'm also concerned about the parallel runs. kuttl allows to set the namespace name instead of using the generated one, so I'm wondering if we could use it somehow to distinguish what namespaces to clean up?

Reply:
No. It runs only once because of concurrency setting

Reply:
https://github.com/percona/everest-operator/blob/b09599c577495224c8f80ef37a861ce53c441587/.github/workflows/ci.yml#L53 check this. It runs only once and the second run won't be triggered until the previous one is finished.